app/lowdefy.yaml

```diff
@@ -1,18 +1,20 @@
-lowdefy: 4.0.0-alpha.23
+lowdefy: 4.0.0-rc.15
 
-config:
-  homePageId: home
-  theme:
-    lessVariables:
-      '@primary-color': '#438ef7'
-      '@border-radius-base': 6px
-      '@border-radius-sm': 4px
-      '@border-color-base': '#69c0ff'
-      'border-width-base': 2px
+# config:
+#   homePageId: home
```

Comment:
Don't we need to define a home page?

Reply:
80a9093d631a3cc914ed4aaa38e2cb6b4cff2ca4